package.json

```diff
@@ -9,6 +9,11 @@
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "flow-coverage": "flow-coverage-report -i \"./src/**/*.js\"",
     "lint": "eslint \"{src,specs,e2e}/**/*.js\" && flow check",
+    "eslint-circle": "eslint 'src/consumer/*.js' 'src/consumer/component/*.js' --format junit -o junit/eslint-results.xml",
+    "eslint-circle-debug": "eslint 'src/consumer/*.js' 'src/consumer/component/*.js'",
+    "flow-coverage-circle": "flow-coverage-report -i 'src/consumer/*.js' -i 'src/consumer/component/*.js' --threshold 70 -t json -o junit",
```

Comment:
is this really a junit? maybe there is a better format to use. need to check it. maybe you can also create an html report, and  just upload it as an artifact

Reply:
It's not. It's JSON format, and CircleCI doesn't know to read it. (according to the docs it knows only to read JSON format of Cucumber).
It does save the JSON as an artifact, which is helpful for debugging.
In any case, I'm probably going to remove it. I found a better tool to handle Flow.